cibuildwheel/windows.py

```diff
@@ -106,21 +89,19 @@ def shell(args, env=None, cwd=None):
     temp_dir = tempfile.mkdtemp(prefix='cibuildwheel')
     built_wheel_dir = os.path.join(temp_dir, 'built_wheel')
 
-    if IS_RUNNING_ON_TRAVIS:
-        # instal nuget as best way for provide python
-        shell(["choco", "install", "nuget.commandline"])
-        # get pip fo this installation which not have. 
-        get_pip_url = 'https://bootstrap.pypa.io/get-pip.py'
-        get_pip_script = 'C:\\get-pip.py'
-        shell(['curl', '-L', '-o', get_pip_script, get_pip_url])
+    # instal nuget as best way for provide python
+    nuget = 'C:\\nuget.exe'
+    download('https://dist.nuget.org/win-x86-commandline/latest/nuget.exe', nuget)
+    # get pip fo this installation which not have.
+    get_pip_script = 'C:\\get-pip.py'
+    download('https://bootstrap.pypa.io/get-pip.py', get_pip_script)
 
     python_configurations = get_python_configurations(build_selector)
     for config in python_configurations:
         config_python_path = get_python_path(config)
-        if IS_RUNNING_ON_TRAVIS and config.nuget_version is not None and not os.path.exists(config_python_path):
-            shell(["nuget", "install"] + get_nuget_args(config))
-            if not os.path.exists(os.path.join(config_python_path, 'Scripts', 'pip.exe')):
-                shell([os.path.join(config_python_path, 'python.exe'), get_pip_script ])
+        simple_shell([nuget, "install"] + get_nuget_args(config))
```

Comment:
Do we need to do for every `config`?

Reply:
yes. we need to install all versions with nuget here. If already installed, nuget just does nothing.

Reply:
Sorry, I got confused looking for the line where `nuget` is installed. But I'm now realizing the executable you download probably doesn't need to be installed anymore but it already ready to be executed.